app/scripts/lib/local-store.js

```diff
@@ -1,18 +1,29 @@
-// We should not rely on local storage in an extension!
-// We should use this instead!
-// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/local
-
 const extension = require('extensionizer')
 const log = require('loglevel')
 
+/**
+ * A wrapper around the extension's storage local API
+ *
+ * Note: We should not rely on local storage in an extension!
+ * We should use this instead! [storage.local]{@linkcode https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/local}
+ *
+ * @deprecated
```

Comment:
Actually this file is the `storage.local` wrapper, and is not deprecated.

Reply:
Thanks, fixed!